lib/reporter.js

```diff
@@ -57,6 +58,14 @@ let buildStackTraceJSON = function(error, projectRoot) {
   })
 }
 
+let normalizePath = path => {
+  return path.replace('file:///', '')                         // Randomly inserted file url protocols
+             .replace(/[/]/g, '\\')                           // Temp switch for Windows home matching
+             .replace(fs.getHomeDirectory(), '~')             // Remove users home dir for apm-dev'ed packages
```

Comment:
There are a lot of places where fsplus.getHomeDirectory() is used - we should clean them all up together.  I'm assuming we can safely drop the node 0.9 stuff now (os.homedir() was introduced in 2.3)